models/_sector/nft/trades/old/platforms/magiceden_solana_events.sql

```diff
@@ -31,120 +32,120 @@ WITH me_txs AS (
     ) AS me_instructions
     FROM {{ source('solana','transactions') }}
     WHERE (
-         array_contains(account_keys, 'M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K') -- magic eden v2
-         OR array_contains(account_keys, 'CMZYPASGWeTz7RNGHaRJfCq2XQ5pYK6nDvVQxzkH51zb')
+         contains(account_keys, 'M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K') -- magic eden v2
+         OR contains(account_keys, 'CMZYPASGWeTz7RNGHaRJfCq2XQ5pYK6nDvVQxzkH51zb')
     )
-    AND success = 'True'
+    AND success = true
     {% if not is_incremental() %}
-    AND block_date > '2022-01-05'
+    AND block_date > TIMESTAMP '2022-01-05'
     AND block_slot > 114980355
     {% endif %}
     {% if is_incremental() %}
     -- this filter will only be applied on an incremental run
-    AND block_date >= date_trunc("day", now() - interval '1 week')
+    AND block_date >= date_trunc('day', now() - interval '7' day)
     {% endif %}
 
 )
 SELECT
   'solana' as blockchain,
   'magiceden' as project,
-  CASE WHEN (array_contains(account_keys, 'M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K')) THEN 'v2'
-  WHEN (array_contains(account_keys, 'CMZYPASGWeTz7RNGHaRJfCq2XQ5pYK6nDvVQxzkH51zb')) THEN 'launchpad_v3'
+  CASE WHEN (contains(account_keys, 'M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K')) THEN 'v2'
+  WHEN (contains(account_keys, 'CMZYPASGWeTz7RNGHaRJfCq2XQ5pYK6nDvVQxzkH51zb')) THEN 'launchpad_v3'
   END as version,
-  signatures[0] as tx_hash,
+  from_base58(signatures[1]) as tx_hash,
```

Comment:
i think we need to align on approach for solana spells. between these spells, the new orca whirlpool spells in PR queue and `prices_tokens`, i think we may be slightly misaligned?
- `prices_tokens` writes in varbinary format, so it can join the cross-chain spell
- orca whirpool spells appear to write as-is, without a function to convert, as seen [here](https://github.com/duneanalytics/spellbook/pull/3707/files#diff-0576be6634b68cdb81185a4376f442e9be34f967a6f182656a63b89c87a1f3d0R162) -- notice that the prices join condition does the opposite conversion
- this spell going back to varbinary approach

cc @andrewhong5297

Reply:
on the NFT side the decision seems simple.
If we want it included in the overall of `nft.trades` it needs to be varbinary, if we're fine excluding solana from the cross chain spells we can keep as varchar.